src/translations/screens/TripSearch.ts

```diff
@@ -195,6 +195,24 @@ const TripSearchTexts = {
       },
     },
   },
+  onboarding: {
+    title: _('Filter i reisesøk 🎉', 'Filters in travel search 🎉'),
+    body: {
+      part1: _(
+        'Nå kan du bruke filter i reisesøk og få reiseforslag tilpasset dine behov.',
+        'You can now use filters in travel searches and get travel suggestions adapted to your needs.',
+      ),
+      part2: _(
+        'Velg transportmiddel, hvor lang tid du ønsker å ha mellom bytter og hvor raskt du beveger deg.',
+        'Choose your means of transport, how long time you prefer between changes and how fast you walk.',
```

Comment:
In the first iteration the filter only includes selecting transport mode, so the text about time between changes and walk speed should not be included now.